### ISSUE TITLE
Message when adding TREZOR account

### ISSUE THREAD

**Issue by mizunashi** [NemProject/NanoWallet#381]
## Expected Behavior
"Account successfully generated." Is displayed.

## Current Behavior
"Account successfully ganerated. Do not forget to down load your updated wallet!" Is displayed.

## Possible Solution
TREZOR Separate processing when using and not using.

## Steps to Reproduce
<! --- Provide an unambiguous set of steps to reproduce this bug. ->
1. Login with TREZOR
2. Go to the Account menu
3. Press + Add a new account info wallet
4. Press add new account
5. "Account successfully ganerated. Do not forget to download your updated wallet!" Is displayed.

# Details
- Donation Address: NCVGXT-CV7YYG-CUTOWR-SEALEV-HVTDFR-J54BQY-DKTI
- NEM Membership ID: NEM-000616

**Comment by AnthonyLaw** [comment]
Do you able to export the second account after you successful Added new Account?

**Comment by cryptoBeliever** [comment]
Agree. The message should be different for Trezor / Ledger since you don't have to back up your wallet file for the hardware wallet. Same in the latest version (2.6.2)
